weather_app/management/commands/utils.py

```diff
@@ -0,0 +1,61 @@
+import os
+import csv
+from weather_app.models import WeatherCharacteristic,Weather,CustomTimeStampModel
+from django.core import exceptions
+
+
+""" reading a files form directory"""
+def reading_files(path):
+    all_files = []
+    """
+    loop getting files from path
+    sending files to the "all_files" list
+    """
+    for root, directory, files in os.walk(path):
+        """ here finding .txt entention files """
+        for file in files:
+            if '.txt' in file:
+                all_files.append(os.path.join(root, file))
+    """ returning the all_files list """
+    return all_files
+
+""" sending data to the database fily by bile """
+def data_sending_to_database(files):
+
+    for f in list(files):
+        with open(f) as csv_file:
+            csv_reader = csv.reader(csv_file, delimiter=',')
```

Comment:
see `DictReader` of `csv` library that will be more optimized for this scenario.